utils/spacewalk-common-channels.ini

```
@@ -3569,6 +3569,53 @@ checksum = sha256
 base_channels = debian-11-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian11-Uyuni-Client-Tools/Debian_11/
 
+[debian-12-pool-amd64-uyuni]
+label    = debian-12-pool-amd64-uyuni
+checksum = sha256
+archs    = amd64-deb
+repo_type = deb
+name     = Debian 12 (bookworm) pool for amd64 for Uyuni
+gpgkey_url =
+gpgkey_id =
+gpgkey_fingerprint =
+repo_url = http://deb.debian.org/debian/dists/bookworm/main/binary-amd64/
+
+[debian-12-amd64-main-updates-uyuni]
+label    = debian-12-amd64-main-updates-uyuni
+name     = Debian 12 (bookworm) AMD64 Main Updates for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-12-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian/dists/bookworm-updates/main/binary-amd64/
+
+[debian-12-amd64-main-security-uyuni]
+label    = debian-12-amd64-main-security-uyuni
+name     = Debian 12 (bookworm) AMD64 Main Security for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-12-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian-security/dists/bookworm-security/updates/main/binary-amd64/
+
```

backports seems to be missing (I see it at https://github.com/uyuni-project/uyuni/pull/7269/files=
```suggestion

[debian-12-amd64-main-backports-uyuni]
label    = debian-12-amd64-main-backports-uyuni
name     = Debian 12 (bookworm) AMD64 Main Backports for Uyuni
archs    = amd64-deb
repo_type = deb
checksum = sha256
base_channels = debian-12-pool-amd64-uyuni
repo_url = http://deb.debian.org/debian/dists/bookworm-backports/main/binary-amd64/
```